lib/nx_gzlib.c

```diff
@@ -225,7 +262,21 @@ int nx_gzread(gzFile file, void *buf, unsigned len)
 	return stream->total_out - last_total_out;
 }
 
-int nx_gzwrite (gzFile file, const void *buf, unsigned len)
+int nx_gzread(gzFile file, void *buf, unsigned len) {
+	return __gzread(file, buf, len, 1);
+}
+
+/** \brief Write to a compressed file. Internal implementation.
+ *
+ *  @param file nx_streamp to be processed.
```

Comment:
pedantic: file is actually a gzip format file descriptor and the internal state have a stream.

Reply:
Modified.